Dockerfile.critical_css

```diff
@@ -0,0 +1,44 @@
+ARG image_tag=latest
+ARG node_version
+FROM pinterb/jq:0.0.16 AS jq
+FROM elifesciences/journal_npm:${image_tag} as npm
+
+FROM node:${node_version}
+
```

Comment:
there was a link this list of packages originally pointed to, for maintenance?

Reply:
Added comment.